ci/do_ci.sh

```diff
@@ -39,15 +39,38 @@ function bazel_with_collection() {
   collect_build_profile $1
 }
 
+function cp_binary_for_outside_access() {
+  DELIVERY_LOCATION=$1
+  cp -f \
+    "${ENVOY_SRCDIR}"/bazel-bin/source/exe/envoy-static \
+    "${ENVOY_DELIVERY_DIR}"/${DELIVERY_LOCATION}
+}
+
+function bazel_sizeopt_binary_build() {
```

Comment:
Can we make a single function and call with params? This seems mostly duplicated from bazel_release_binary_build